compiler-rt/test/rtsan/blocking_call.cpp

```diff
@@ -28,7 +21,7 @@ int main() {
   return 0;
   // CHECK-NOT: {{.*safe_call*}}
   // CHECK: ==ERROR: RealtimeSanitizer: blocking-call
-  // CHECK-NEXT: Call to blocking function `custom_blocking_function` in real-time context!
+  // CHECK-NEXT: Call to blocking function `custom_blocking_function()` in real-time context!
   // CHECK-NEXT: {{.*custom_blocking_function*}}
```

Comment:
Would it be wise to add `// CHECK-NOT: {{.*In blocking function.*}}` (or whatever the correct regex is to make sure the printf doesn't happen inside `custom_blocking_function`?